.lintballrc.json

```diff
@@ -0,0 +1,41 @@
+{
+    "check_args": {
```

Comment:
why do we introduce lintball?

Reply:
to format generated code, currently I have only enabled commands.py,  otherwise, we have to deal with all kinds of empty line and line too long because of the template's natural feature.